### PR TITLE
feat: few acs for stop orders in auction

### PR DESCRIPTION
**Pull request by Sohill-Patel**
split existing acs that are actually two cases, since the current tests only cover one case
Added a couple acs for stop orders in auction / opening auction

### CODE REVIEW

File: protocol/0014-ORDT-order_types.md [hunk 1]
```diff
@@ -370,8 +372,15 @@ In Spot market, for multiple iceberg orders submitted as a batch of orders with
...
-- A stop order placed either prior to or during an auction, where the uncrossing price is within the triggering range, will immediately execute following uncrossing. (<a name="0014-ORDT-066" href="#0014-ORDT-066">0014-ORDT-066</a>)
+- A stop order placed during an auction will not execute during an auction, nor will it participate in the uncrossing. (<a name="0014-ORDT-065" href="#0014-ORDT-065">0014-ORDT-065</a>)
+- A stop order placed during an auction, where the uncrossing price is within the triggering range, will immediately execute following uncrossing. (<a name="0014-ORDT-066" href="#0014-ORDT-066">0014-ORDT-066</a>)
+
+- A stop order placed prior to an auction will not execute during an auction, nor will it participate in the uncrossing. (<a name="0014-ORDT-134" href="#0014-ORDT-134">0014-ORDT-134</a>)
+- A stop order placed prior to an auction, where the uncrossing price is within the triggering range, will immediately execute following uncrossing. (<a name="0014-ORDT-135" href="#0014-ORDT-135">0014-ORDT-135</a>)
```
Comment (davidsiska-vega) [inline protocol/0014-ORDT-order_types.md:379]: Is this different to 0014-ORDT-066?

**Review by davidsiska-vega — CHANGES_REQUESTED** [review]
One question there...

File: protocol/0014-ORDT-order_types.md (hunk 1 above)
Comment (Sohill-Patel) [reply protocol/0014-ORDT-order_types.md:379]: looking at the feature tests 0014-ORDT-066 and 0014-ORDT-065 we only run the scenario when stop order is created during auction. 
I created these two acs to cover the scenario when stop order is created prior to an auction

Comment (davidsiska-vega) [reply protocol/0014-ORDT-order_types.md:379]: I see it - one is during one is prior!

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**